server/src/index.ts

```diff
@@ -18,11 +18,11 @@ app.use(cors());
 app.use(bodyParser.json());
 app.use("/flexpa-access-token", flexpaAccessToken);
 app.use("/fhir", fhirRouter);
-app.use("/", (_req, res) => {
+app.use("/healthcheck", (_req, res) => {
```

Comment:
@ellipsis-dev can you change this so that the handler is in its own file that is imported like the others?